app/views/books/show.html.erb

```diff
@@ -1,30 +1,41 @@
-<div class="wrapper">
-  <div class="image"></div>
-  <div class="book-title"><%= book.title %></div>
-  <div class="book-author"><%= book.author %></div>
-  <br>
-  <div class="book-author book-year">
-    <span><%= book.year %> </span>
-    <%= link_to 'Request', book_path(book), class:"request" %>
+<div class="comments-wrapper">
+  <div class="book-container">
+    <div class="image"></div>
+    <div>
+      <div class="book-title"><%= book.title %></div>
+      <div class="book-author"><%= book.author %></div>
+      <br>
+      <div class="book-year-request">
+        <h4><%= book.year %></h4> 
+        <% if !current_user.requested?(book) %>
+          <span id='<%= book.id %>'>
+            <%= link_to 'Request', requests_path(book_id: book.id), method: :post, remote: true, class: "request" %>
+          </span>
+        <% else %>
+          <span class="<% if current_user.request_status(book) == "denied"%>denied bold<% else %>bold dark-opacity big-20<% end %>"><%= current_user.request_status(book) %></span>  
```

Comment:
Use an inline `if`.

Also, the class `bold` is always used, so you can exclude it from the conditional.